provider/resources.go

```diff
@@ -2868,6 +2868,20 @@ func Provider() *tfbridge.ProviderInfo {
 					}
 					return config, nil
 				},
+				TransformFromState: func(ctx context.Context, inputs resource.PropertyMap) (resource.PropertyMap, error) {
+					if _, ok := inputs["db_name"]; ok {
+						return inputs, nil
+					}
+					name, ok := inputs["name"]
+					if !ok {
+						// It's seems like inputs are not in a valid state
+						return inputs, nil
+					}
+					tfbridge.GetLogger(ctx).Debug(`migrating from "name" to "dbName"`)
+					inputs["dbName"] = name
```

Comment:
Can you explain what's the difference between inputs["db_name"] and inputs["dbName"]?

Should not this always be "dbName" in Pulumi states?

Reply:
Your right. That should be `dbName`.